catroidTest/src/org/catrobat/catroid/uitest/formulaeditor/FormulaEditorEditTextTest.java

```diff
@@ -133,6 +133,14 @@ public void testSingleTapOnFunctionName() {
 				.getSelectionEnd());
 	}
 
+	public  void testHighlightColor(){
+		BackgroundColorSpan colorSpan=new BackgroundColorSpan(0xff33b5e5);
+		BackgroundColorSpan colorSpan1=(BackgroundColorSpan) Reflection.getPrivateField(
+				new FormulaEditorEditText(getActivity()),"COLOR_HIGHLIGHT");
+		assertTrue("Highlight color is wrong",colorSpan.getBackgroundColor()==colorSpan1.getBackgroundColor());
+
+	}
+
```

Comment:
Please reformat your code!  ("Ctrl + Alt + L" in Android Studio) 

There should be a whitespace before and after the `=` operator and a whitespace after the `,` parameter delimiter .
Furthermore use some meaningful variablenames such as `editTextColorSpan` instead of `colorSpan1`.